pkgs/development/libraries/opencl-clhpp/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "opencl-clhpp";
-  version = "2.0.15";
+  version = "2.0.17";
 
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "OpenCL-CLHPP";
-    rev = "v${version}";
-    sha256 = "sha256-A12GdevbMaO2QkGAk3VsXzwcDkF+6dEapse2xfdqzPM=";
+    rev = "v2022.05.18"; # XXX: tag uses different name than current release
```

Comment:
Is this the new version format? If so please update version accordingly.

Reply:
The [release page](https://github.com/KhronosGroup/OpenCL-CLHPP/releases/tag/v2022.05.18) shows the version as v2.0.17, but pointing to the tag v2022.05.18.